pkg/types/baremetal/platform.go

```diff
@@ -34,6 +34,7 @@ type Host struct {
 	HardwareProfile string           `json:"hardwareProfile"`
 	RootDeviceHints *RootDeviceHints `json:"rootDeviceHints,omitempty"`
 	BootMode        BootMode         `json:"bootMode,omitempty"`
+	NetworkConfig   string           `json:"networkConfig,omitempty"`
```

Comment:
Please add documentation here for this field so that it shows up in the `openshift-install explain` output.